test/chpldoc/basics/irregularCommentClose.doc.chpl

```diff
@@ -0,0 +1,6 @@
+module IrregularCommentClose {
+  /*
+     This module's comment close is on its own line.
+   */
```

Comment:
Did you mean for this comment to be before the `module` declaration? It is not used in the chpldoc output.

Reply:
I can't recall now if it needed to be below it to cause the error I was seeing. I'll investigate and update in the next PR.